src/addresses/types.ts

```diff
@@ -1,19 +1,19 @@
 import type { RequestOptions, RequestPayload } from '../types';
 
-export enum AddressPurpose {
+export enum AddressPurposes {
```

Comment:
why are we changing these?

Reply:
> The PR also fixes a bug where the AddressPurposes type name has been reverted to what it originally was since this attribute is used in extension and app.